phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexExpressionIT.java

```diff
@@ -123,7 +126,7 @@ protected void helpTestCreateAndUpdate(boolean mutable, boolean localIndex) thro
                     + (localIndex ? "LOCAL" : "")
                     + " INDEX IDX ON "
                     + fullDataTableName
-                    + " ((UPPER(varchar_pk) || '_' || UPPER(char_pk) || '_' || UPPER(varchar_col1) || '_' || UPPER(char_col2)),"
+                    + " ((UPPER(varchar_pk) || '_' || UPPER(char_pk) || '_' || UPPER(varchar_col1) || '_' || UPPER(b.char_col2)),"
```

Comment:
This was not required, I was just testing that it worked with the column family name.